### DISCUSSION TITLE
Few errors in the log.

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#1242] [Q&A]
Hi,

I've just installed opensnitch on a new system ( sysvinit, Kernel 6.6.63 )

I wiped the log and restarted the system to check if everything was good.

- _IMP  Start writing logs to /var/log/opensnitchd.log_ ✔️
- _WAR  error starting ebpf monitor method: &{eBPF error when enabling kprobes: cannot write "p:ptcp_v6_connect tcp_v6_connect\n" to kprobe_events: write /sys/kernel/debug/tracing/kprobe_events: no such file or directory 1}_

  _WAR  error starting ebpf monitor method: &{eBPF error when enabling kprobes: cannot write "p:ptcp_v6_connect tcp_v6_connect\n" to kprobe_events: write /sys/kernel/debug/tracing/kprobe_events: no such file or directory 1}
ERR  Reconf() -> Init() error: &{1 eBPF error when enabling kprobes: cannot write "p:ptcp_v6_connect tcp_v6_connect\n" to kprobe_events: write /sys/kernel/debug/tracing/kprobe_events: no such file or directory}_

  _WAR  Unable to set new process monitor (ebpf) method from disk: eBPF error when enabling kprobes: cannot write "p:ptcp_v6_connect tcp_v6_connect\n" to kprobe_events: write /sys/kernel/debug/tracing/kprobe_events: no such file or directory_

    - is this caused by https://github.com/evilsocket/opensnitch/issues/1239#issuecomment-2543638997 or because I don't have ipv6 ? ( `CONFIG_IPV6=n` )
    &nbsp;
- _IMP  [nftables] AddSystemRules() fw disabled_

  _WAR  nftables: error applying changes: conn.Receive: netlink receive: operation not supported_

  _ERR  Error while adding interception tables: nftables: error adding system firewall table: mangle, family: inet (1)_

  - ```shell
    #/etc/opensnitchd/default-config.json
    "Firewall":"nftables"
    ```
    ```shell
    #/etc/opensnitchd/system-fw.json
    "Enabled":false
    ```
    I do not want that opensnitch touch my nftable rules.
    So I suppose this is good , and it's not actually an ERR ? (or is it something wrong here ? )

So the main question is will opensnich will catch the connections ?

Thanks

**Comment by gustavo-iniguez-goya** [comment]
hey @BobSquarePants,

For the first problem, see if `/sys/kernel/debug/tracing` is mounted, if is anything in there.

For the second problem, see if you can list or add rules with nft.You should have at least 2 rules in "inet mangle output" table, and "inet filter input".
 Also see if you have this option configured in the kernel: CONFIG_NFT_CT=m

**Comment by BobSquarePants** [comment]
Hey @gustavo-iniguez-goya,

1. yes `/sys/kernel/debug/tracing` is mounted (2135 directory and 10354 files )

2. - `nft list ruleset` works
    - I don't have any `inet` table, as I don't use ipv6, but I do have a `ip` table
    For my `ip` table I do have a `type filter hook input priority 0; policy drop;` rule
    ~~in opensnitch 1.6.0rc5 I didn't have also and mangle table and it's was working fine...~~
    - yes `CONFIG_NFT_CT` is set as `m`
    and `lsmod | grep conn` return
    > nf_conntrack   xxxxx 1 nft_ct
    > nf_defrag_ipv4 xxxx 1 nf_conntrack

Thanks for your help btw !

**Comment by BobSquarePants** [comment]
I close this discussion as I recreated it with better formatting --> https://github.com/evilsocket/opensnitch/discussions/1258
